lib/chef/application.rb

```diff
@@ -153,9 +153,13 @@ def apply_extra_config_options(extra_config_options)
     end
 
     def set_specific_recipes
```

Comment:
while you're here can we get a YARD comment added to this method

Reply:
sure, I will add YARD comment for this method.